com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1996,8 +2001,13 @@ public IResource getResource(String category, String key, boolean mustExist) thr
         if (val != null && val.trim().length() > 0) {
             resource = this.getResource(val);
         }
-        if (mustExist && resource == null) {
-            throw new IOException(String.format("Resource does not exist: '%s'  (%s.%s)", resource.getAbsPath(), category, key));
+        if (mustExist) {
+            if (resource == null) {
+                throw new IOException(String.format("Resource is null: %s.%s = '%s'", category, key, val==null?"null":val));
```

Comment:
Previously, `resource` was null, and generated _another_ error, in the middle of this error reporting.